README.md

```diff
@@ -1,3 +1,4 @@
 # TicTacToe_Ruby_v2
 
-[![Build Status](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2.svg?branch=master)](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2)
+
+[![Build Status](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2.svg?branch=convert_to_gem)](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2)
```

Comment:
The typical use case of the build status is to communicate off the home page what's going on with master. If actively someone is working on a feature branch then they're probably just going to CI directly and looking at what's going on there. The build status is sort of a "quick check" to make sure that the master branch on this project is always green (should never ever be red). It's not as evil when other branches temporarily go red, because nobody is (nobody should be) depending on them.